src/utils/api.ts

```diff
@@ -33,6 +34,22 @@ export class ApiService {
     this.app.get("/", (_req: Request, res: Response) => {
       res.send("🐮 Moooo!");
     });
+    this.app.get("/health", async (_req: Request, res: Response) => {
+      // Using an iterator, process all the chain contexts, storing the health
+      // in a map, and if any of the contexts are unhealthy, return false
```

Comment:
maybe shorter imperative comment? 

`Check the overall health. It's healthy if every network is healthy`

Reply:
Done.